vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -550,11 +551,28 @@ void AdsImpl::SetConfirmationsIsReady(
   is_confirmations_ready_ = is_ready;
 }
 
-std::map<uint64_t, std::vector<AdsHistory>> AdsImpl::GetAdsHistory() {
+std::map<uint64_t, std::vector<AdsHistory>> AdsImpl::GetAdsHistory(
+  const Ads::AdHistoryFilterType adHistoryFilterType) {
   std::map<uint64_t, std::vector<AdsHistory>> ads_history;
   base::Time now = base::Time::Now().LocalMidnight();
 
   auto ad_history_details = client_->GetAdsShownHistory();
+
+  std::unique_ptr<AdHistoryFilter> adHistoryFilter = nullptr;
```

Comment:
Minor nit: The `unique_ptr` will automatically initialize to `nullptr`.

Reply:
`adHistoryFilter` should use `snake_case`

Reply:
Spotted - I thought I'd fixed them all... sigh

Reply:
> Minor nit: The `unique_ptr` will automatically initialize to `nullptr`.

Np, will address :+1:

Reply:
Done